Tests/test_GenBank.py

```diff
@@ -3192,92 +3194,84 @@ class GenBankTests(unittest.TestCase):
 
     def test_invalid_product_line_raises_value_error(self):
         """Parsing invalid product line."""
-        def parse_invalid_product_line():
-            rec = SeqIO.read(os.path.join('GenBank', 'invalid_product.gb'),
-                             'genbank')
-        self.assertRaises(ValueError, parse_invalid_product_line)
+        path = os.path.join('GenBank', 'invalid_product.gb')
+        self.assertRaises(ValueError, SeqIO.read, path, 'genbank')
```

Comment:
Good spot - I probably missed the more elegant usage of ``assertRaises`` with arguments when that was written.

However, why define the path variable?

Reply:
Setting the path is not part of the test for ValueError, so it's better to keep it out.